samples/azure/asb-and-asbs-side-by-side/ASBAndASBSSideBySide/ASBAndASBSSideBySide/SideBySide.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace ASBAndASBSSideBySide
+{
+    extern alias TransportASBS; //  Edit The Project And See The Reference 
+    using NServiceBus;
+
+    public class SideBySide
+    {
+        public SideBySide()
+        {
+            var configurationASB = new EndpointConfiguration("ASB");
+            var transportASB = configurationASB.UseTransport<NServiceBus.AzureServiceBusTransport>();
+
+            transportASB.Queues().EnableBatchedOperations(true);
+
+            var configurationASBS = new EndpointConfiguration("ASBS");
+            var transportASBS = configurationASBS.UseTransport<TransportASBS::NServiceBus.AzureServiceBusTransport>();
+
+            // transportASBS.PrefetchMultiplier(1);
```

Comment:
Don't comment it out. It's good that both transports can be configured using configuration API specific to their packages. I.e. ASB has `.Queues()` where ASBS has `PrefetchMultiplier()`.